src/azul/openapi/schema.py

```diff
@@ -192,16 +196,62 @@ def enum(*items: PrimitiveJSON, type_: TYPE = None) -> JSON:
             "bar"
         ]
     }
-    """
 
+    >>> assert_json(enum(items=['a']))
+    {
+        "type": "string",
+        "enum": [
+            "a"
+        ]
+    }
```

Comment:
```suggestion
    }
    
```
Add similar blank lines between all the newly added doctests.